app/src/androidTest/java/ch/epfl/sdp/blindly/match/ProfileTest.kt

```diff
@@ -0,0 +1,69 @@
+package ch.epfl.sdp.blindly.match
+
+import androidx.test.espresso.matcher.ViewMatchers.assertThat
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.hamcrest.Matchers.contains
+import org.hamcrest.Matchers.equalTo
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@HiltAndroidTest
+class ProfileTest {
+
+    @get:Rule
+    var hiltRule = HiltAndroidRule(this)
+
+    @Before
+    fun setup() {
+        hiltRule.inject()
+    }
+
+    val andre1 = Profile(name = "André", age =  25)
+    val andre2 = Profile(name = "André", age = 24)
+    val pierre1 = Profile(name = "Pierre", age = 24)
+    val pierre2 = Profile(name = "Pierre", age = 25)
```

Comment:
Same comment here. Btw wouldn't it be clearer if andre1.age = pierre1.age instead if andre1.age = pierre2.age ?